### PR TITLE
Finish autocomplete feature

### PR DESCRIPTION
**Pull request by jakelauer27**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,85 @@
+{
+  "parser": "babel-eslint",
```
Comment (libbyeh) [inline .eslintrc]: Thank you for walking me through this code. Its great to see the binary tree in action!

File: src/Characters.js [hunk 2]
```diff
@@ -2,177 +2,169 @@ import React, { Component } from 'react';
...
+    super();
+    this.state = {
+      characterList: [],
+      card: '',
+      characters: [],
+      currentCharacter: '',
+      universe: 'all',
+      suggestions: []
+    };
+    this.trie = new Trie();
```
Comment (libbyeh) [inline src/Characters.js]: Pretty straight on adding to the constructor and populating in componentDidMount.

**Comment by libbyeh** [comment]
As mentioned, thanks for representing team SmashNerd in the autocomplete lesson and then walking me through what you learned. Much appreciated!

File: src/App.js [hunk 3]
```diff
@@ -42,13 +41,13 @@ class App extends Component {
...
           this.keys.map( key => {
             if (this.state[key]) {
-              switch(key) {
-                case 'Characters':
-                return <Characters key={key} />
-                case 'Stages':
-                return <Stages key={key} />
-                case 'Compare':
-                return <Compare key={key}/>
+              switch (key) {
```
Comment (JustinTorrance) [inline src/App.js]: Very cool. I'm excited to see this in action.

File: src/App.js [hunk 4]
```diff
@@ -42,13 +41,13 @@ class App extends Component {
...
+              case 'Characters':
```
Comment (JustinTorrance) [inline src/App.js]: Everything looks solid. No suggested edits from me.

**Pull request merged**